src/topology/algebra/const_mul_action.lean

```diff
@@ -114,6 +115,19 @@ instance {ι : Type*} {γ : ι → Type*} [∀ i, topological_space (γ i)] [Π
   [∀ i, has_continuous_const_smul M (γ i)] : has_continuous_const_smul M (Π i, γ i) :=
 ⟨λ _, continuous_pi $ λ i, (continuous_apply i).const_smul _⟩
 
+lemma is_compact.smul {α β} [has_smul α β] [topological_space β]
+  [has_continuous_const_smul α β] (a : α) {s : set β}
+  (hs : is_compact s) : is_compact (a • s) := hs.image (continuous_id'.const_smul a)
+
+instance nnreal.has_continuous_const_smul
+  {α} [mul_action ℝ α] [topological_space α]
+  [has_continuous_const_smul ℝ α] : has_continuous_const_smul nnreal α :=
+{ continuous_const_smul := λ c,
+  begin
+    convert continuous_id'.const_smul (↑c : ℝ),
+    apply_instance
+  end }
```

Comment:
I guess you should wait for #16713 ...

Reply:
Yes, it seems that it's just about to merge and then I will pull the latest `master` commits to this branch and remove my own instance.